ert3/console/_console.py

```diff
@@ -1,17 +1,30 @@
-import ert3
-
 import argparse
-from pathlib import Path
+import pathlib
```

Comment:
This import seems unnecessary as only the Path symbol is used, which is also imported below.